### ISSUE TITLE
Include explanatory text outside of data fields (TBC / post-user testing)

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#572]
via @plscully 
RANDOM THINGS to keep in mind as we finalize the new site
**Placeholders v. Labels in form fields** -- I’m concerned that using placeholders instead of labels for hints/instructions on how to complete the form fields will impede our goal of collecting accurate data. Many of our fields introduce new concepts to users, so we need to make it as simple as possible for them to access and understand the questions we’re asking them to respond to. We will also be developing a glossary to which we can link some of the more complex and/contested terms.  I did some looking around for advice about labels v. placeholders, and [this article](https://www.nngroup.com/articles/form-design-placeholders/) rings true with me. Its bottom line advice is to keep both the label and the instructions/hint outside the form field.

**Comment by jesicarson** [comment]
Related: https://github.com/participedia/usersnaps/issues/90

**Comment by jesicarson** [comment]
@andreadelrio Placeholder draft for related content - 

H1: Related Content
Explanatory text: Related content refers to cases, methods or organizations that are directly connected or subordinate to this particular case. If the content has not yet been published on Participedia, it won't show up in the fields below, but you can use quick submit to add it. 

H2: Cases
Explanatory text: Related cases include public participation events that are part of a larger initiative. For example, this case could be considered a component part of another case or vice versa.  
Box text: Begin typing the title of an existing case
(TBC Plus icon - opens a new window to quick submit a case?)

H2: Methods
Explanatory text: Related methods include participatory methods or tools used during this particular case. 
Box text: Begin typing the title of an existing method
(TBC Plus icon - opens a new window to quick submit a method?)

H2: Organizations
Explanatory text: Related organizations include any organization that played a direct role in the planning or execution of this particular case. 
Box text: Begin typing the title of an existing organization
(TBC Plus icon - opens a new window to quick submit an organization?)

**Comment by scottofletcher** [comment]
@jesicarson Those placeholders are really excellent! One small change I would make is to H2: replace the word 'case' with 'initiative' or 'engagement' - 'case' is merely short for 'case study' (which has its own problems since we're looking for documentary/encyclopedic entries and not analyses...). I would change the text to: "Related case entries include public participation events that are part of a larger initiative. For example, this event could be a component of a broader initiative or vice versa."

Also, I think you're right - the main issue I'm having (and that I think other users will have) is with the word "Related". Here's some alternatives:
- Relevant Content ('relevant' makes it clear we only want those cases/orgs/methods that are _directly_ connected to this entry as opposed to 'related' which implies 'similar' or 'like')
- Corresponding Content -or- Corresponding Material ('corresponding' would include both the methods and organizations _directly_ used/involved in this initiative as well as case entries on separate but connected 'component' events)

**Comment by jesicarson** [comment]
Right, but we want to be clear that the "broader initiative" is actually another case on the website. Need to be consistent in terminology.  

"Corresponding Content" could work..

**Comment by scottofletcher** [comment]
That's an ideal though - I think we may find a lot of authors trying to add components that don't have their own PPedia entries yet. Perhaps we could do something similar to the methods and organizations boxes which allow you to quick submit?

**Comment by jesicarson** [comment]
That function isn’t implemented yet, but yes we should prompt authors to create the related/corresponding content if it’s not yet on ppedia. I’ll update the text above to reflect that.

**Comment by scottofletcher** [comment]
Great! I think its important not to limit authors to entering only those cases/methods/orgs to those that already exist in PPedia - otherwise we're going to be missing part of the picture (eg. what if a case has another component?). Is it possible to have the box text reflect that (ie. get rid of the 'existing' or make it clear that it will auto-fill if it exist and, if not, that you can quick submit it)?

**Comment by jesicarson** [comment]
Going to make mockups. See #658.
